### DISCUSSION TITLE
Can someone recommend an ESP32 board?

### DISCUSSION THREAD

**Discussion by patrickaroo** [FortySevenEffects/arduino_midi_library#271] [Q&A]
I've had luck in the past just using a board for any particular library that someone else has used successfully. Can anyone suggest an esp32 board they've used that has worked for this library? Thanks!

PS - I'm also curious about non-esp32 boards that have worked.

**Comment by alkurop** [comment]
Hi. I'm using esp32-s2 soala. It works after connecting a usb-c external output to pins with D- D+, and also VCC to 5v, while the rest of the stuff connects to vcc 3.3v, because esp32 controls voltage on that pin.
I also tried esp32 s1, but it doesn't supporting connecting as a usb device to computer.
Curious about esp32-s3 now.
